libs/coin-tester/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '2.4'
+
+services:
+  speculos:
+    # Speculos isn't supported for Mac M1 yet. You have to build the image locally as described in the README.
+    # Otherwise you can use the official image
+    image: ghcr.io/ledgerhq/speculos:latest
```

Comment:
We should look to have an env variable here to chose the right imagine depending on M1 or CI